wire-ios/Wire-iOS/Sources/AnalytitcsServiceConfigurationBuilder.swift

```diff
@@ -33,7 +33,7 @@ struct AnalyticsServiceConfigurationBuilder {
         return AnalyticsServiceConfiguration(
             secretKey: secretKey,
             serverHost: countlyURL,
-            didUserGiveTrackingConsent: !ExtensionSettings.shared.disableAnalyticsSharing
+            didUserGiveTrackingConsent: !(ExtensionSettings.shared.disableAnalyticsSharing ?? true)
```

Comment:
maybe we should avoid to rely on the fallback value, for example making the isAnalyticsDisabled static this would be possible

```suggestion
            didUserGiveTrackingConsent: !TrackingManager.isAnalyticsDisabled
```

Reply:
I'd like to avoid a static accessor, Swift concurrency won't like it. There's some strange dependency relationships here that we need to solve (but I would defer it for later): TrackingManager and the ExtensionSettings should really go to WireDomain where they can be used more directly.